Makefile

```diff
@@ -136,6 +141,22 @@ clean:
 	$(RM) -r build
 	$(RM) -r vendor
 
+# File target that generates a diff file if formatting is incorrect
+$(FORMAT_CHECK): $(GO_FILES)
```

Comment:
Right off the initial clone and make, it is seen that existing merged go files are attempted to fix go formatting by this change, e.g.   dialout/dialout_client/dialout_client_test.go etc. So, after make, those files are in the 'modified' status in git, which could be inconvenient for community users to work on the repo.
Why can't we just fix the format for those files to merge?